src/mock-doc/element.ts

```diff
@@ -237,6 +258,95 @@ patchPropAttributes(MockScriptElement.prototype, {
   type: String,
 });
 
+export class MockDOMMatrix {
+  static fromMatrix() {
+    return new MockDOMMatrix();
+  }
+  a: number = 1;
+  b: number = 0;
+  c: number = 0;
+  d: number = 1;
```

Comment:
I had this question in the [original implementation](https://github.com/ionic-team/stencil/pull/3408#discussion_r891194336) that I think still stands

> For my own understanding, why do `a` and `d` get assigned a value of 1, but the other single letter properties of the matrix are assigned a value of 0?

Reply:
Oh sorry, I missed responding to this one! I don't know. I simply used the default values returned when a DOMMatrix is instantiated (see screenshot).
<img width="363" alt="Screen Shot 2022-06-13 at 11 18 30 AM" src="https://user-images.githubusercontent.com/92547717/173398979-5ed94c46-21da-4f17-aaea-4d81862b9700.png">